flatpak/org.mamedev.MAME.yaml

```diff
@@ -0,0 +1,80 @@
+id: org.mamedev.MAME
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: mame
+separate-locales: false
+finish-args:
+  - --device=all
+  - --persist=.mame
+  - --filesystem=home
```

Comment:
That's really unfortunate, a solution should be found: either dropping it, should work in most cases.
If that's just because of the `history` folder, `historypath` could be changed to another location.

Reply:
If you meant `--filesystem=home` that's for loose File Manager in TAB menu for consoles and computers, among other things (base path for fonts?), `-verbose` is your friend.

Reply:
Could it be in read-only then?

Reply:
The `--filesystem=home` line is just the flatpak sandbox permissions for mame to be able to create its own `~/.mame` directory.  The first run of the flatpak should be `mame -createconfig` which will generate `~/.mame/mame.ini` and that configuration can survive flatpak upgrades and even be used outside of the flatpak in a native install.